UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx

```diff
@@ -369,4 +369,60 @@
     <value>Dictionary value if exists</value>
     <comment>property description</comment>
   </data>
-</root>
+  <data name="Authentication_Category" xml:space="preserve">
```

Comment:
remove multiple duplicates, leave the first entry earlier in the file

e.g.
lines 148-151 and 372-375, remove the section for 372-375 (duplicate)